src/Router.js

```diff
@@ -11,28 +15,80 @@ const LandingPage = lazy(() => import('./Components/LandingPage/LandingPage'));
 const CreateImageWizard = lazy(() =>
   import('./Components/CreateImageWizard/CreateImageWizard')
 );
+const INVENTORY_TOTAL_FETCH_URL = '/api/inventory/v1/hosts';
 
 export const Router = () => {
+  const [hasSystems, setHasSystems] = useState(true);
+  const [selected, setSelected] = useState(false);
+  useEffect(() => {
+    try {
+      axios
+        .get(`${INVENTORY_TOTAL_FETCH_URL}?page=1&per_page=1`)
+        .then(({ data }) => {
+          setHasSystems(data.total > 0);
+        });
+    } catch (e) {
+      /*eslint-disable no-console*/
+      console.log(e);
+      /*eslint-enable no-console*/
+    }
+  }, [hasSystems]);
   const edgeParityFlag = useFlag('edgeParity.image-list');
   return (
-    <Routes>
-      <Route path="*" element={<LandingPage />}>
-        <Route path="imagewizard/:composeId?" element={<CreateImageWizard />} />
-        <Route path="share/:composeId" element={<ShareImageModal />} />
-      </Route>
+    <Suspense
```

Comment:
I don't think that I want to do this at the <Router>... maybe we can move this to <LandingPage> instead?